spec/libraries/helpers_spec.rb

```diff
@@ -61,10 +61,11 @@
     end
   end
 
-  context 'debian 7 32 bit' do
+  # rubocop:disable ChefDeprecations/NodeSetWithoutLevel
```

Comment:
@tas50 hmm, something to fix?

Reply:
Yeah, need to refactor the test so they run quicker and without setting all the node attributes. Next PR or so